csharp_dotnetcore/9.Message-Routing/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+
+namespace MessageRoutingBot
+{
+    /// <summary>
+    /// Entry point for Bot application.
```

Comment:
Remove comments from this class, in alignment with what the ASP.Net Core team does.

Reply:
Sounds good.